backends/qualcomm/passes/annotate_and_quant_scalar.py

```diff
@@ -37,24 +38,22 @@ class AnnotateAndQuantScalar(ExportPass):
     ]
     quant_attrs_key = "quant_attrs"
 
-    def __init__(self):
+    def __init__(self, edge_program: torch.export.ExportedProgram):
         super(AnnotateAndQuantScalar, self).__init__()
+        self.edge_program = edge_program
 
     def _get_source_scalar_node(self, node: torch.fx.Node) -> torch.fx.Node:
         """
         This recursion function is specific for multiply followed by a cast
         """
-        if node.op == "get_attr":
+        if node.op == "placeholder":
```

Comment:
Scalar arguments should really be "burnt" into the node and I dont expect them to be placholder (and get_attr) nodes. Is that not what you see?

Reply:
No, it will be translated into placeholder after IR capture. The lifting part is introduce by https://github.com/pytorch/pytorch/blob/main/torch/_export/passes/lift_constant_tensor_pass.py